src/services/ui/e2e/tests/home/index.spec.ts

```diff
@@ -1,21 +1,33 @@
 import { test, expect } from "@playwright/test";
-import * as $ from "@/selectors";
+
+const password = process.env.VITE_BOOTSTRAP_USERS_PW!;
 
 test("has title", async ({ page }) => {
   await page.goto("/");
-
-  // Expect a title "to contain" a substring.
   await expect(page).toHaveTitle(/CMS MAKO/);
 });
 
-test("get issues link", async ({ page }) => {
-  const navSelectors = new $.NavSelectors(page);
+test("has faq Page", async ({ page }) => {
+  await page.goto("/");
+  await page.getByRole("link", { name: "FAQ" }).click();
+});
+
+test("log in test", async ({ page }) => {
   await page.goto("/");
+  await page.getByRole("button", { name: "Sign In" }).click();
+  await page.getByRole("textbox", { name: "name@host.com" }).click();
+  await page.getByRole("textbox", { name: "name@host.com" }).fill("george@example.com");
```

Comment:
lets put this email value somewhere else, we might be able to hardcode it in the playwright config file so we can use it like a env variable